<launch>

<!--
 <node pkg="urg_node" type="urg_node" name="urg_node"/>
-->
  <node pkg="selfie_park" type="park_server" name="park_server" output="screen">
    <param name="ackermann_topic" value="/sim_drive"/>
    <param name="odom_topic" value="/odom"/>
  </node>

  <node pkg="selfie_parking" type="parking_node" name="search_server" output="screen">
    <param name="/point_min_x" value="-0.5"/>
    <param name="/point_max_x" value="1.4"/>
    <param name="/point_min_y" value="-0.4"/>
    <param name="/point_max_y" value="0.1"/>

    <param name="distance_to_stop" value="0.2"/>

    <param name="visualization_type" value="13"/>

    <param name="scans_to_ignore_when_stopped" value="3"/>
    <param name="scans_taken" value="2"/>
  </node>

  <node pkg="selfie_perception" type="detect_obstacles" name="selfie_obstacle_detector">
    <param name="visualize" value="true" />
    <param name="max_range" value="2.0"/>
    <param name="min_range" value="0.1"/>
    <param name="line_search_max_range_difference_" value="0.04"/>
    <param name="line_search_max_slope_difference_" value="2.0"/>
    <param name="line_search_min_slope_difference_" value="0.05"/>
    <param name="line_search_slope_difference_ratio_" value="0.10"/>
    <param name="line_search_min_length_" value="0.05"/>
    <param name="line_min_length_" value="0.05"/>
    <param name="obstacle_nominal_length_" value="0.4"/>   
  </node>

</launch>
